omnibus/resources/inspec/deb/conffiles.erb

```diff
@@ -0,0 +1,3 @@
+<% config_files.each do |file| -%>
```

Comment:
This and the other `deb` and `rpm` resource files appear identical to the default versions that omnibus will use.  Perhaps we can remove them from here?